### DISCUSSION TITLE
Different keybindings than which-key.nvim

### DISCUSSION THREAD

**Discussion by holisticxyz** [LazyVim/LazyVim#1283] [Q&A] [answered]
I'm a beginner to NeoVim. Wanted to try it out. After install I started to try out some keybindings. On [this](https://www.lazyvim.org/keymaps#general) page, there is a `Search word under cursor` which should be available through `gw` but in LazyVim it's actually available with `*`. Did I missed anything?

**Comment by dpetka2001** [comment, marked as answer]
`*` is the default builtin keybinding of vim/neovim for search word under cursor. Lazyvim just added another keybinding doing the same thing based on the maintainer's intuition. See `:h star`.

**Reply by dpetka2001** [reply]
Basically there is a small difference. `*` searches for the first occurrence of the word under the cursor, but does not take into consideration the word under the cursor and puts the cursor to the next occurrence. `gw` takes into consideration the word under cursor and highlights it as the first result without moving the cursor to the first occurrence after the current word. `gw` is mapped to `*N`.

**Comment by folke** [comment]
fyi: I just removed the gw keymap from LazyVim
